### PR TITLE
Register warden Dimitar2828

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden Dimitar2828

### CODE REVIEW

File: _data/handles/Dimitar2828.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "handle": "Dimitar2828",
+  "link": "",
```
Comment (Dimitar32) [inline _data/handles/Dimitar2828.json:3]: Hello

**Review by Dimitar32 — COMMENTED** [review]
Hello. I cant understand whats wrong with my registration

**Comment by SamusElderg** [comment]
> Hello. I cant understand whats wrong with my registration

Have you checked into the `i-want-to-be-a-warden` channel in the Code4rena Discord? What is your global Discord username/handle?

**Comment by SamusElderg** [comment]
Can not find a matching user in Discord, will leave the application open for now though (see above communication with the user via GitHub)

**Comment by Dimitar32** [comment]
> Can not find a matching user in Discord, will leave the application open for now though (see above communication with the user via GitHub)

my username is dimitar333 #9326

**Comment by Dimitar32** [comment]
> > Can not find a matching user in Discord, will leave the application open for now though (see above communication with the user via GitHub)
> 
> my username is dimitar333 #9326

when I want to log in code4rena it says me "registration pending"

**Comment by SamusElderg** [comment]
> > > Can not find a matching user in Discord, will leave the application open for now though (see above communication with the user via GitHub)
> > 
> > 
> > my username is dimitar333 #9326
> 
> when I want to log in code4rena it says me "registration pending"

You need to check into the `i-want-to-be-a-warden` channel in the Code4rena Discord server. Mention this PR (Dimitar2828) when you do so we can cross reference it with your discord name from there

**Comment by SamusElderg** [comment]
Closing PR for now, will re-open when user checks into Discord with matching local server nickname

**Pull request closed without merging**